src/components/Bets/BetRow.tsx

```diff
@@ -50,13 +58,13 @@ export const BetRow: React.FC<Props> = ({ config, betData, onClick }) => {
         {moment.unix(betData.blockNumber).fromNow() ?? <Skeleton />}
       </td>
       <td className="px-2 py-4 truncate">
-        {utils.formatting.parseBytes16String(betData.marketId) ?? <Skeleton />}
-      </td>
-      <td className="px-2 py-4 truncate">
-        {utils.formatting.parseBytes16String(betData.propositionId) ?? (
+        {`${raceDetails.date} ${raceDetails.location} Race ${raceDetails.raceNumber}` ?? (
           <Skeleton />
         )}
       </td>
+      <td className="px-2 py-4 truncate">
+        {`Horse ${winningPropositionId} win` ?? <Skeleton />}
```

Comment:
when `winningPropositionId` is undefined
it will just show `Horse  win` instead of skeleton

maybe better like this?
```
winningPropositionId ? `Horse ${winningPropositionId} win` : <Skeleton />
```

Reply:
+1 good catch

Reply:
oh yep thats a good idea!